### ISSUE TITLE
FlexibleColumnLayout: Nested interactive controls are not announced by screen readers

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents-react#1708]
**Description**
axe DevTools shows these two errors for the FlexibleColumnLayout component

**Isolated Example**
```
<FlexibleColumnLayout
          accessibilityTexts={
            { 
            startArrowRightText: i18n.getText("ONBOARDING_EXPAND_FIRST_COLUMN"), 
            startArrowleftText: i18n.getText("ONBOARDING_COLLAPSE_FIRST_COLUMN") 
            }
         }
...
```

**Axe tool suggested fix:**
Element has focusable descendants

Also no matter the value of startArrowRightText and startArrowleftText , it alway shows "Expand the first column" and "Collapse the first column"

**Screenshots**

<img width="1917" alt="Screen Shot 2021-06-09 at 2 08 12 PM" src="https://user-images.githubusercontent.com/65705637/121430774-5e6f5d00-c92d-11eb-872e-1f67c5209a72.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:~0.31.9
`@ui5/webcomponents-react` version: ~0.131.7
Operating System: MacOS
Browser: Chrome

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
